src/SPC/command/DownloadCommand.php

```diff
@@ -368,4 +371,50 @@ private function findPreBuilt(array $assets, string $filename): ?string
         }
         return null;
     }
+
+    /**
+     * @throws RuntimeException
+     */
+    private function _clean(): int
+    {
+        logger()->warning('You are doing some operations that not recoverable: removing directories below');
+        logger()->warning(SOURCE_PATH);
+        logger()->warning(DOWNLOAD_PATH);
+        logger()->warning(BUILD_ROOT_PATH);
+        logger()->alert('I will remove these dir after 5 seconds !');
+        sleep(5);
+        if (PHP_OS_FAMILY === 'Windows') {
+            f_passthru('rmdir /s /q ' . SOURCE_PATH);
+            f_passthru('rmdir /s /q ' . DOWNLOAD_PATH);
+            f_passthru('rmdir /s /q ' . BUILD_ROOT_PATH);
+        } else {
+            f_passthru('rm -rf ' . SOURCE_PATH . '/*');
+            f_passthru('rm -rf ' . DOWNLOAD_PATH . '/*');
+            f_passthru('rm -rf ' . BUILD_ROOT_PATH . '/*');
+        }
+        return static::FAILURE;
+    }
+
+    private function getDefaultAlternativeSource(string $source_name): array
+    {
+        return [
+            'type' => 'custom',
+            'func' => function (bool $force, array $source, int $download_as) use ($source_name) {
+                logger()->debug("Fetching alternative source for {$source_name}");
+                // get from dl.static-php.dev
+                $url = "https://dl.static-php.dev/static-php-cli/deps/spc-download-mirror/{$source_name}/?format=json";
+                $json = json_decode(Downloader::curlExec(url: $url, retry: intval(getenv('SPC_RETRY_TIME') ? getenv('SPC_RETRY_TIME') : 0)), true);
```

Comment:
haha, I'm just renaming this env variable.

Also `intval(getenv('SPC_RETRY_TIME') ? getenv('SPC_RETRY_TIME')` should be `(int)getenv('SPC_DOWNLOAD_RETRIES') ?? 0`